packages/moonstream-components/src/components/CryptoUnicorns/LootboxCard.tsx

```diff
@@ -15,26 +15,29 @@ const _LootboxCard = ({
   lootboxBalance,
   showQuantity = true,
   isVideo = false,
+  grayedOut = false,
   ...props
 }: {
   displayName: string;
   imageUrl: string;
   lootboxBalance: number;
   showQuantity?: boolean;
   isVideo?: boolean;
+  grayedOut?: boolean;
 }) => {
   return (
     <Flex {...props} pb={10}>
       <VStack maxW="250" border="solid" borderColor="#373E9B" borderRadius="lg">
         <Image
           src={imageUrl}
           as={isVideo ? "video" : undefined}
+          filter={grayedOut ? "grayscale(100%)" : undefined}
+          opacity={grayedOut ? "0.3" : undefined}
```

Comment:
Good to define "0.3" as a constant - as we create more badges, that will be the better approach.

Reply:
This will work for other badges. The card is used for everything.